EVS changes for Android 10

arminn opened the pull request:
EVS changes for Android 10 

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
Fix evs build for Android 10 VNDK

It is impossible to use libgui during
VNDK build due to incompatibility.
Replacing it and fixing the Android 10 API difference.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

rshym commented on sampleDriver/Android.bp:
```diff
@@ -26,33 +30,37 @@ cc_binary {
+        "libEGL",
+        "libGLESv2",
+        "libhardware",
+                "libpng",
```
It could be better to remove extra spaces.

arminn committed:
evs manager: Rename specific implementation to *xt

Service evs_manager is already defined by the generic component.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs app: Update to the revision android-10.0.0_r3

Use evs app from packages/servce/Car/evs based on
revision android-10.0.0_r3

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs app: Use vendor config

Do not rely on system based config files.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs app: Make app message more verbose

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

rshym commented on app/evs_app.cpp:
```diff
@@ -97,7 +97,7 @@ int main(int argc, char** argv)
     // Load our configuration information
     ConfigManager config;
-    if (!config.initialize("/vendor/etc/vehicle/evs_app_xt.json")) {
+    if (!config.initialize("/system/etc/automotive/evs/config.json")) {
```
Seems like same change is duplicated in next commit "evs app: Use vendor config".

arminn committed:
evs app: Do not use Pixel copy renderer

Pixel copy renderer won't help without started
SurfaceFlinger(which imply GL ready).
So there is no point in non-GL solutions for now.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs app: Define evs app as service

This will give evs app correct launch
permissions and it will be disabled by
default. I.e. user can start it using:
start evs_app_xt

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs app: Increase max inflight frames to 4

Sometimes, 2 inflight frames is not enough for
correct work.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs app: Print configuration file name

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs driver: Increase frame count to 4

For better user experience and stability,
increasing frame count to 4.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs driver: WA: Place binary onto the system partition

Due to link conflicts with libgui(not available for vendor),
placing evs driver to the system partition.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

arminn committed:
evs driver: Add V4L2_PIX_FMT_XBGR32 format support

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>
